apps/andi/lib/andi_web/live/edit_live_view.ex

```diff
@@ -220,12 +238,13 @@ defmodule AndiWeb.EditLiveView do
         false -> "Saved successfully. You may need to fix errors before publishing."
       end
 
-    changeset =
```

Comment:
I moved this to be part of the complete_validation stuff on line 215. NO IDEA why it was like this, but it was stomping on form changes